src/internal/connector/exchange/contact_test.go

```diff
@@ -19,25 +20,39 @@ func TestContactSuite(t *testing.T) {
 }
 
 func (suite *ContactSuite) TestContactInfo() {
+	initial := time.Now()
+
 	tests := []struct {
 		name         string
 		contactAndRP func() (models.Contactable, *details.ExchangeInfo)
 	}{
 		{
 			name: "Empty Contact",
 			contactAndRP: func() (models.Contactable, *details.ExchangeInfo) {
-				i := &details.ExchangeInfo{ItemType: details.ExchangeContact}
-				return models.NewContact(), i
+				var contact = models.NewContact()
```

Comment:
Nit: avoid `var` unless initializing zero-valued variables.
```suggestion
				contact := models.NewContact()
```